gateway/src/apicast/policy/rate_limiting_to_service/apicast-policy.json

```diff
@@ -0,0 +1,53 @@
+{
+  "$schema": "http://apicast.io/policy-v1/schema#manifest#",
+  "name": "rate limiting to service policy",
+  "description": ["This policy adds rate limiting to the entire Service."],
+  "version": "builtin",
+  "configuration": {
+    "type": "object",
+    "properties": {
+      "limitters": {
+        "description": "List of limitters to be applied",
+        "type": "array",
+        "items": {
+          "type": "object",
+          "properties": {
+            "limitter": {
+              "description": "Limitter class name",
+              "type": "string"
+            },
+            "key": {
+              "description": "Key of Redis or Name of lua_shared_dict shm zone",
+              "type": "string"
+            },
+            "values": {
+              "description": "Values to be applied",
+              "type": "array",
+              "items": {
+                "type": "integer"
```

Comment:
This is pretty non descriptive and won't have really useful UI.

IMO we could use `anyOf` and write a description for each limiter and describing each field (`burst`, `rate`, `delay`, ...).